src/components/PasswordResetFlow/ResetPage/index.js

```diff
@@ -0,0 +1,136 @@
+                ariaInvalid={!!errors.password}
+                inputClassName="create-account-input-text"
+                labelClassName={
+                  errors.password
+                    ? 'create-account-errors-field'
+                    : 'create-account-input-field'
+                }
+                showPlaceholder={false}
+              />
+              <div className="info-circle">
+                <button
+                  type="button"
+                  onClick={openPasswordModal}
+                  className="info-circle-button"
+                  aria-labelledby="password-info"
+                />
+              </div>
+              <ToolTipModalContainer
+                modalIsOpen={passwordModalIsOpen}
+                closeModal={closeModal}
+                label="show-password-info-dialog"
+                content="Salasanassa tulee olla vähintään 10 merkkiä, ja siinä pitää
+                  olla isoja kirjaimia, pieniä kirjaimia ja numeroita."
+              />
+            </div>
+            <div className="error-text">
+              {errors.password &&
+                errors.password.type === 'required' &&
+                'Kirjoita salasana'}
+              {errors.password &&
+                errors.password.type === 'pattern' &&
+                'Salasanan on oltava vähintään 10 merkkiä pitkä ja siinä pitää olla isoja kirjaimia, pieniä kirjaimia ja numeroita.'}
+              {errors.password &&
+                errors.password.type === 'maxLength' &&
+                'Salasanan on oltava enintään 30 merkkiä pitkä.'}
+            </div>
+          </div>
+
+          <div className="formfield-container">
+            <ValidatedInputField
+              label="Salasana uudestaan"
+              name="passwordConfirm"
+              ref={register({
+                required: true,
+                validate: value => {
+                  return value === watch('password')
+                },
+              })}
+              type="password"
+              ariaInvalid={!!errors.passwordConfirm}
+              inputClassName="create-account-input-text"
+              labelClassName={
+                errors.passwordConfirm
+                  ? 'create-account-errors-field'
+                  : 'create-account-input-field'
+              }
+              showPlaceholder={false}
+            />
+            <div className="error-text">
+              {errors.passwordConfirm &&
+                errors.passwordConfirm.type === 'required' &&
+                'Kirjoita salasana uudestaan'}
+              {errors.passwordConfirm &&
+                errors.passwordConfirm.type === 'validate' &&
+                'Salasanat eivät ole samat'}
+            </div>
+          </div>
+          <button
+            type="submit"
+            className="create-account-button"
+            onKeyPress={handleSubmit(handleNewPassword)}
+            tabIndex="0"
+          >
+            {'Kirjaudu '}
```

Comment:
I think this should be something else, like "Vaihda salasana"

Reply:
I agree, I left it as "Kirjaudu" because it was labeled as that in the design. Should I change it?

Reply:
I think it's like that in design by mistake from copy-pasting buttons from previous views. We can change it. 👍

Reply:
I changed the text to "Vaihda salasana" 😊